word_guess.rb

```diff
@@ -0,0 +1,198 @@
+require 'colorize'
+
+class WordGuess
+	attr_accessor :guesses_wrong, :win
+
+	def initialize
```

Comment:
It seems like the indentation in this method is the opposite. The `def` should be over one to the left, lines 7-12 should be one to the right, and the `end` on line 13 should be one to the left as well. There are a few other opportunities throughout this file to adjust accordingly. Let me know if you have any questions about the indentation!